src/main/javascript/view/partials/signature.js

```diff
@@ -573,7 +573,13 @@ SwaggerUi.partials.signature = (function () {
       value = _.cloneDeep(value);
       var example = value.examples;
       if (_.isString(example)) {
-        example = jsyaml.safeLoad(example);
+        try {
+          example = jsyaml.safeLoad(example);
```

Comment:
In my pull request I added it to index.html so you don't get undefined jsyaml errors.